src/app.ts

```diff
@@ -9,8 +10,35 @@ import './apps/transaction';
 
 @customElement('gc-trade-app')
```

Comment:
This element is going to be used only for standalone app, have draft WIP .. In the main app tx center and notif center are going to be at root level and only trade app will be routed ..  Anyway we share the same idea of passing api addr and acc ;)